bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/WeatherApi.java

```diff
@@ -107,6 +107,11 @@ public Collection<NAMain> getFavoriteAndGuestStationsData() throws NetatmoExcept
         return answer != null ? answer.getElements() : List.of();
     }
 
+    public Collection<NAMain> getOwnStationsData() throws NetatmoException {
```

Comment:
Can you rename method `getOwnedStationData` to `getOwnStationData` for consistency (line 94)?

Reply:
Improved this.